gulpfile.js

```diff
@@ -146,14 +146,14 @@ gulp.task( 'uglify', [ 'requirejs' ], function() {
 
 // ----- css ----- //
 
-var minifyCSS = require('gulp-minify-css');
+var cleanCSS = require('gulp-clean-css');
 
 gulp.task( 'css', function() {
   gulp.src('css/flickity.css')
     // copy to dist
     .pipe( gulp.dest('dist') )
     // minify
-    .pipe( minifyCSS({ advanced: false }) )
+    .pipe( cleanCSS({ advanced: false }) )
```

Comment:
Thanks for this! Didn't know about the deprecation. As this is a separate feature, I would have submitted this bit of code in a separate PR, so it doesn't get hung up in this one.